usaspending_api/search/tests/data/spending_by_award_test_data.py

```diff
@@ -0,0 +1,197 @@
+import pytest
+
+from model_mommy import mommy
+from usaspending_api.awards.models import Award
+from usaspending_api.references.models.ref_program_activity import RefProgramActivity
+
+
+@pytest.fixture
+def spending_by_award_test_data():
+
+    mommy.make("references.LegalEntity", legal_entity_id=1001)
```

Comment:
Are these three LegalEntity mocks necessary?

Reply:
Guess we'll find out in a minute.

Those were legacy code.  I just moved the data creation to a new file so I could share it rather than mock up essentially the same code again.

Tests worked locally, so I'm feeling pretty good that they weren't required.